### PR TITLE
Govi Escudero

### PR DESCRIPTION
**Pull request by jaklundun**
## Tasks
* [ ] Objects Task
* [ ] Arrays Task
* [ ] Callbacks Task 
* [ ] Stetch Task

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,34 +63,68 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+//console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+console.log(inventory[32]);
```
Comment (enitchals) [inline assignments/arrays.js:68]: You did log out the year/make/model, but not in the correct format. But based on your work on Challenge 2, I think you could complete this one using a string using a [template literal.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

File: assignments/callbacks.js [hunk 2]
```diff
@@ -1,30 +1,93 @@
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
+
+//callbackOne  for firstItem()
+function callbackOne (arr) {
```
Comment (enitchals) [inline assignments/callbacks.js:5]: Very thorough with your callback functions here! :+1:

**Review by enitchals — COMMENTED** [review]
Great use of comments to show what you're doing! In the professional world, you'll mostly be making comments for other people... but as a student, these comments can really help you if you decide to come back and reference your old code for another project.

**Comment by jaklundun** [comment]
Thank you, Ellen!

Made the update you suggest.

**Pull request closed without merging**